src/Components/PaperEditor/ProgrammeScriptContainer/index.js

```diff
@@ -90,6 +90,23 @@ const ProgrammeScriptContainer = (props) => {
 
       createPaperEdits(paperEdit);
       setSaved(true);
+    };
+  };
+
+  const handleClearProgrammeScript = () => {
+    const confirmDelete = window.confirm('Are you sure you want to delete?');
+    if (confirmDelete) {
+      const newList = [
+        {
+          type: 'insert',
+          text: 'Insert point to add selection',
+        }
+      ];
+      setElements(newList);
+      setResetPreview(true);
+
+      handleSaveProgrammeScript(newList);
+      props.trackEvent({ category: 'paperEditor programmeScript', action: 'handleDelete all elements' });
```

Comment:
Should the action be called `handleClearProgrammeScript` as the others use this convention?

Reply:
Thanks!